src/components/themed/AddressTile.js

```diff
@@ -73,12 +74,21 @@ class AddressTileComponent extends React.PureComponent<Props, State> {
   componentDidMount(): void {
     this._setClipboard(this.props)
     this.props.addressTileRef(this)
+    if (this.props.isCameraOpen) {
+      this.handleScan()
+    }
   }
 
   componentWillUnmount(): void {
     this.props.addressTileRef(undefined)
   }
 
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.isCameraOpen !== this.props.isCameraOpen && this.props.isCameraOpen) {
```

Comment:
this can simplify into:
`if (this.props.isCameraOpen && !prevProps.isCameraOpen)`